lib/srv/desktop/rdp/rdpclient/Cargo.toml

```diff
@@ -7,19 +7,28 @@ edition = "2018"
 [lib]
 crate-type = ["staticlib"]
 
+[profile.dev]
+debug = 1
+lto = "off"
+
+[profile.release]
+debug = 1
+lto = "thin"
+codegen-units = 1
```

Comment:
Why 1?

Reply:
https://doc.rust-lang.org/cargo/reference/profiles.html

On debug it disables some very verbose symbol generation. 1 is all that's needed for things like stacktraces.
Setting codegen-units to 1 generally improves performance. The only case it is detrimental is if you have one huge monocrate since it will only parallelize across dependencies.

Reply:
No I meant for `codegen-units`.

Reply:
So that will prevent Rust from splitting the crate into CGU's that are compiled separately. This means you generally get better performance at the expense of a smidge slower compile times. This degradation however only really matters if you are bound by your main crate and not your dependencies for compilation. As most of our compile time is spent on dependencies this doesn't impact build times measurably (during my testing) while allowing better code optimization.